stable/sysdig/README.md

```diff
@@ -177,6 +177,32 @@ And then, upgrade Helm chart with:
 $ helm upgrade my-release -f values.yaml stable/sysdig
 ```
 
+## How to upgrade to the last version
+
+In case you deployed the chart with a values.yaml file, you just need to modify (or add if it's missing) the `image.tag` field and execute:
+
+```bash
+$ helm install --name sysdig -f values.yaml stable/sysdig
+```
+
+If you deployed the chart setting the values as CLI parameters, like for example:
+
+```bash
+$ helm install \
+    --name sysdig \
+    --set sysdig.accessKey=xxxx \
+    --set secure.enabled=true \
+    --set ebpf.enabled=true \
+    --namespace sysdig-agent \
+    stable/sysdig
+```
+
+You will need to execute something like:
```

Comment:
You will need to execute:

There's no need about `something like`, feels to vage :)